soc/nordic/nrf53/Kconfig

```diff
@@ -138,6 +139,13 @@ config SOC_NRF_GPIO_FORWARDER_FOR_NRF5340
 	help
 	  hidden option for including the nRF GPIO pin forwarding
 
+config SOC_NRF53_CPUNET_MGMT
+	bool
+	default y
```

Comment:
nit: there is no need to both make this option `default y` and also `select` it above

Reply:
Removed `default y`